Dialog/RoomConfigDialog.cpp

```diff
@@ -0,0 +1,227 @@
+#include "RoomConfigDialog.h"
+#include "ui_RoomConfigDialog.h"
+
+RoomConfigDialog::RoomConfigDialog(QWidget *parent) :
+    QDialog(parent),
+    ui(new Ui::RoomConfigDialog)
+{
+    ui->setupUi(this);
+}
+
+RoomConfigDialog::~RoomConfigDialog()
+{
+    delete ui;
+}
+
+RoomConfigDialog::InitDialog()
+{
+    InitComboBoxItems();
+    ui->ComboBox_AlphaBlendAttribute->setCurrentIndex(0);
+    ui->ComboBox_Layer0MappingType->setCurrentIndex(0);
+    // TODO
+
+}
+
+RoomConfigDialog::InitDialog(DialogParams::RoomConfigParams CurrentRoomParams)
```

Comment:
This function should have a struct pointer as the parameter, because otherwise the program must push all the values of the struct onto the stack which is slower and uses more memory than pushing a pointer to the stack for the function call.

Reply:
done